[2024-12-28 21:49:18,938] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: agile_data_science_batch_prediction_model_training.pyspark_train_classifier_model 2024-12-28T21:26:24.162472+00:00 [queued]>
[2024-12-28 21:49:18,953] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: agile_data_science_batch_prediction_model_training.pyspark_train_classifier_model 2024-12-28T21:26:24.162472+00:00 [queued]>
[2024-12-28 21:49:18,954] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-12-28 21:49:18,954] {taskinstance.py:1096} INFO - Starting attempt 4 of 4
[2024-12-28 21:49:18,955] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-12-28 21:49:18,962] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): pyspark_train_classifier_model> on 2024-12-28T21:26:24.162472+00:00
[2024-12-28 21:49:18,966] {standard_task_runner.py:52} INFO - Started process 195 to run task
[2024-12-28 21:49:18,971] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'agile_data_science_batch_prediction_model_training', 'pyspark_train_classifier_model', '2024-12-28T21:26:24.162472+00:00', '--job-id', '28', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/setup.py', '--cfg-path', '/tmp/tmp1vtni6dh', '--error-file', '/tmp/tmpbvw4zv8h']
[2024-12-28 21:49:18,975] {standard_task_runner.py:77} INFO - Job 28: Subtask pyspark_train_classifier_model
[2024-12-28 21:49:19,015] {logging_mixin.py:109} INFO - Running <TaskInstance: agile_data_science_batch_prediction_model_training.pyspark_train_classifier_model 2024-12-28T21:26:24.162472+00:00 [running]> on host 79189d879fe3
[2024-12-28 21:49:19,069] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=agile_data_science_batch_prediction_model_training
AIRFLOW_CTX_TASK_ID=pyspark_train_classifier_model
AIRFLOW_CTX_EXECUTION_DATE=2024-12-28T21:26:24.162472+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-28T21:26:24.162472+00:00
[2024-12-28 21:49:19,070] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2024-12-28 21:49:19,071] {subprocess.py:63} INFO - Running command: ['bash', '-c', '\nspark-submit --master local[8]   /opt/***/practica_creativa//resources/train_spark_mllib_model.py   /opt/***/practica_creativa/']
[2024-12-28 21:49:19,078] {subprocess.py:74} INFO - Output:
[2024-12-28 21:49:19,079] {subprocess.py:78} INFO - bash: line 1: spark-submit: command not found
[2024-12-28 21:49:19,080] {subprocess.py:82} INFO - Command exited with return code 127
[2024-12-28 21:49:19,090] {taskinstance.py:1463} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1165, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1283, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1313, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/bash.py", line 176, in execute
    raise AirflowException('Bash command failed. The command returned a non-zero exit code.')
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code.
[2024-12-28 21:49:19,092] {taskinstance.py:1513} INFO - Marking task as FAILED. dag_id=agile_data_science_batch_prediction_model_training, task_id=pyspark_train_classifier_model, execution_date=20241228T212624, start_date=20241228T214918, end_date=20241228T214919
[2024-12-28 21:49:19,142] {local_task_job.py:151} INFO - Task exited with return code 1
[2024-12-28 21:49:19,178] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
